cargo-shuttle/src/lib.rs

```diff
@@ -603,15 +590,27 @@ impl Shuttle {
             }
         });
 
-        println!("{resources}");
+        let resources = response
+            .resources
+            .into_iter()
+            .map(resource::Response::from_bytes)
+            .collect();
+
+        println!("{}", get_resources_table(&resources));
 
         let addr = if run_args.external {
-            std::net::IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0))
+            Ipv4Addr::new(0, 0, 0, 0)
         } else {
-            Ipv4Addr::LOCALHOST.into()
+            Ipv4Addr::LOCALHOST
         };
+        let addr = SocketAddr::new(addr.into(), run_args.port);
 
-        let addr = SocketAddr::new(addr, run_args.port);
+        println!(
+            "    {} {} on http://{}\n",
+            "Starting".bold().green(),
+            service_name,
+            addr
+        );
```

Comment:
```suggestion
        println!(
            "{:>12} {} on http://{}\n",
            "Starting".bold().green(),
            service_name,
            addr
        );
```
This just makes it easier to keep things aligned without having to count spaces.

Reply:
Same as other comment